server_admin/topics/authentication/webauthn.adoc

```diff
@@ -0,0 +1,197 @@
+
+NOTE: Whether WebAuthn's operations succeeds depends on a user's WebAuthn supporting authenticator, browser and platform. If you use this WebAuthn support, please clarify to what extent those entities supports WebAuthn specification.
+
+The major restriction is as follows :
+
+- A user can register only one WebAuthn supporting Authenticator (called WebAuthn authenticator here)
+- On registering the WebAuthn authenticator, its https://www.w3.org/TR/webauthn/#attestation-statement[Attestation Statement] is not verified
+- Only Two-Factor Authentication (2FA) scenario is supported
+
+==== Setup
+
+The setup procedure of WebAuthn support for 2FA is the following :
+
+Enable User Registration::
+
+An administrator carries out the following operations on `Admin Console` :
+
+- open `Realm Settings -> Login` tab
+- set `User registration` switch ON and push `Save` button
+
+Enable Webauthn Authenticator Registration::
+
+An administrator carries out the following operations on `Admin Console` :
+
+- open `Authentication -> Required Actions` tab
+- push `Register` button
+- select `Webauthn Register` as `Required Action`
+- mark `Enabled` and `Default Action` checkbox
+
+Enable 2FA by Webauthn Authenticator::
+
+An administrator carries out the following operations on Admin Console :
+
+- open `Authentication -> Flows` tab
+- copy and create the new browser flow
+- add `WebAuthn Authenticator` execution below `User Password Form` execution
+- set `OPTIONAL` to `WebAuthn Authenticator` execution's requirement
+- open `Authentication -> Bindings` tab
+- select the created new browser flow as `Browser Flow`
+- push `Save` button
+
+==== Register WebAuthn Authenticator
+
+There are two types of registering a WebAuthn authenticator, for a user who has not yet registered their account on {project_name}, and for a user who has already registered their account on {project_name}.
+
+For the user to identify their registered WebAuthn authenticator afterwards, the user can enter the text as their label.
+
+User without their account::
+
+A user carries out the following operations :
+
+- open the login form
+- click `Register` link
+- fill in items on the register form and push `Register` button
+- the user's browser asks the user to register their WebAuthn authenticator
+- after successful registration, the user's browser asks the user to enter the text as their just registered WebAuthn authenticator's label
+
+User with their account::
+
+At first, an administrator carries out the following operations on `Admin Console` to require the user to register their WebAuthn authenticator after their login :
+
+- open `Users -> (user) -> Details` tab
+- set `WebAuthn Register` on `Required User Actions`
+- push `Save` button
+
+After that, the user carries out the following operations :
+
+- open the login form
+- fill in items and push `Login` button
```

Comment:
Fill in items and click Save.

Reply:
Thank you. I'll fix this sentence as you suggested.